flask/tests/conftest.py

```diff
@@ -97,9 +139,6 @@ def test_database(client):
 
     wes = DatasetType(dataset_type="WES")
     wgs = DatasetType(dataset_type="WGS")
-    db.session.add(wes)
-    db.session.add(wgs)
-    db.session.flush()
```

Comment:
Since you're dropping the two hardcoded cases in favour of the entire array above, you should remove the two variables above too and refer to the literal string where they are used below.